product_docs/docs/epas/17/epas_rel_notes/index.mdx

```diff
@@ -0,0 +1,35 @@
+---
+navTitle: Release notes
+title: "EDB Postgres Advanced Server release notes"
+navigation:
+- epas17_2_0_rel_notes
+
+---
+
+EDB Postgres Advanced Server 17.2 is built on open-source PostgreSQL 17.2, which introduces myriad enhancements that enable databases to scale up and scale out in more efficient ways. 
+
+The EDB Postgres Advanced Server documentation describes the latest version of EDB Postgres Advanced Server 17.2 including minor releases and patches. These release notes provide information on what was new in each release. 
+
+| Version                        | Release date | Upstream merges                                                                                                  |
+|--------------------------------|--------------|------------------------------------------------------------------------------------------------------------------|
+| [17.2](epas17_2_0_rel_notes)   | 21 NOV 2024  | [17.2](https://www.postgresql.org/docs/17/release-17-2.html)                                                     |
```

Comment:
Not sure if we need to correct the name of the file or the link here. I am correction the link in this suggestion. 
Also, the postgres page 17.2 doesn't exist yet. But I guess it will exist at some some point? 
```suggestion
| [17.2](epas17_2_rel_notes)   | 21 Nov 2024  | [17.2](https://www.postgresql.org/docs/17/release-17-2.html)                                                     |
```